src/Behat/Behat/Output/Node/EventListener/AST/OutlineListener.php

```diff
@@ -33,52 +33,15 @@
  */
 final class OutlineListener implements EventListener
 {
-    /**
-     * @var OutlinePrinter
-     */
-    private $outlinePrinter;
-    /**
-     * @var ExamplePrinter
-     */
-    private $examplePrinter;
-    /**
-     * @var StepPrinter
-     */
-    private $stepPrinter;
-    /**
-     * @var SetupPrinter
-     */
-    private $stepSetupPrinter;
-    /**
-     * @var SetupPrinter
-     */
-    private $exampleSetupPrinter;
-    /**
-     * @var ExampleNode
-     */
-    private $example;
+    private ?ExampleNode $example;
```

Comment:
Some properties need to be defined as nullable. I used the opportunity to also change several properties to use constructor promotion